src/main/haskell/kore/src/Kore/Exec.hs

```diff
@@ -348,24 +349,26 @@ prove limit definitionModule specModule = do
         (rewriteAxioms, simplifier, substitutionSimplifier) =
             axiomsAndSimplifiers
     specAxioms <-
-        simplifyRewriteAxioms tools
+        mapM (sequence . fmap (simplifyRewriteRule tools))
```

Comment:
Could you extract the "sequence . fmap (simplifyRewriteRule tools)" or something similar to a separate function that has a signature?